lib/cursor-position-view.coffee

```diff
@@ -49,13 +51,18 @@ class CursorPositionView extends HTMLElement
     atom.workspace.getActiveTextEditor()
 
   updatePosition: ->
-    if position = @getActiveTextEditor()?.getCursorBufferPosition()
-      @row = position.row + 1
-      @column = position.column + 1
-      @goToLineLink.textContent = @formatString.replace('%L', @row).replace('%C', @column)
-      @classList.remove('hide')
-    else
-      @goToLineLink.textContent = ''
-      @classList.add('hide')
+    return if @viewUpdatePending
+
+    @viewUpdatePending = true
+    atom.views.updateDocument =>
```

Comment:
Keeping a reference to this Disposable would be more complete here. If this element is destroyed while this document update is still pending, it might throw exceptions on things like accessing `@classList`.